webonary-cloud-api/README.md

```diff
@@ -0,0 +1,83 @@
+# Webonary Cloud API (WCA)
+Webonary Cloud API provides a way for dictionary data from [FieldWorks (FLex)](https://github.com/sillsdev/FieldWorks) to be stored in AWS Cloud and accessed by applications, including [Webonary (2.0)](https://www.webonary.org).
+
+Complete list of API calls are found in [Webonary Cloud API Guide](./API.md).
+## Architecture
+WCA is a serverless, cloud-based architecture consisting of:
+1. AWS API Gateway
+2. AWS S3
+3. AWS Lambda (Typescript)
+4. MongoDB Altas
+5. (Optional) AWS Certificate Manager ans Route 53 
+
+## Technology Stack
+1. AWS Cloud Development Kit (CDK) for infrastructure provisioning
+2. Typescript for Lamda and CDK
+3. AWS Serverless Application Model (SAM) for local Lambda testing
+4. Jest for unit testing
+5. (Recommended) Visual Studio Code
+
+## Installation and Prerequisites
+1. Clone this [repository](https://github.com/sillsdev/webonary.git) using [git](https://git-scm.com/).
+   1. Note that this is a monorepo, containing code for Webonary Wordpress site, as well as WCA. If you are interested in only WCA, you might consider using [git spare-checkout](https://github.blog/2020-01-17-bring-your-monorepo-down-to-size-with-sparse-checkout/).
```

Comment:
[git sparse-checkout] not spare. :-)

Reply:
Sorry to bother you with all these misspellings :-( I found a good [spell checker for VSCode ](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker) and will use it and include it in .vscode config file.